src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -489,4 +506,14 @@ class ClusterMonitorActor(val cluster: Cluster,
   private def getDbClusterStatus: Future[ClusterStatus] = {
     dbRef.inTransaction { _.clusterQuery.getClusterStatus(cluster.id) } map { _.getOrElse(ClusterStatus.Unknown) }
   }
+
+  private def recordMetrics(origStatus: ClusterStatus, finalStatus: ClusterStatus): IO[Unit] = {
+    for {
+      endTime <- IO(System.currentTimeMillis)
+      name = s"ClusterMonitor/${origStatus}->${finalStatus}"
```

Comment:
not sure if `->` is valid in a metrics name

Reply:
Heh maybe not -- will try it out, and maybe replace with `_to_`